user-manager-service/src/main/java/com/crmfoodestablishment/userauthservice/usermanager/entity/User.java

```diff
@@ -0,0 +1,55 @@
+package com.crmfoodestablishment.userauthservice.usermanager.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.UUID;
+
+@Entity
+@Table(name = "users")
+@Getter
+@Setter
+@NoArgsConstructor
+public class User {
+
+    @Id
+    @SequenceGenerator(name="users_seq", sequenceName = "users_seq")
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "users_seq")
```

Comment:
Why did u decide to use sequences instead of identity?

Reply:
Cause configuration of sequence easier comparing to migration from identity if application will face performance issues

Reply:
Could u pls explain in details  what the problem is? 

Just a reminder that we ae talking abt such max numbers:
The system must be able to store at least 100 employees.
The system must be able to store at least 10000 clients.
The system must be able to store at least 1000 dishes.

What performance degradation are u expecting to see?

Reply:
resolved